test/SerilogWeb.Test/Global.asax.cs

```diff
@@ -7,6 +9,12 @@ public class Global : System.Web.HttpApplication
     {
         protected void Application_Start(object sender, EventArgs e)
         {
+            ApplicationLifecycleModule.FormDataLoggingLevel = LogEventLevel.Debug;
+            ApplicationLifecycleModule.LogPostedFormData = LogPostedFormDataOption.OnMatch;
+            ApplicationLifecycleModule.ShouldLogPostedFormDataPredicate = context => context.Response.StatusCode >= 400;
+
+            ApplicationLifecycleModule.RequestFilterPredicates.Add(context => context.Request.QueryString["nolog"] == "true");
```

Comment:
Is the predicate inverted here? Seems like `!= "true"` would do what I expect, or am I misreading it?